### ISSUE TITLE
ReferenceError: window is not defined

### ISSUE THREAD

**Issue by m-abs** [coolaj86/isbnjs#9]
``` bash
ReferenceError: window is not defined
    at /opt/data-importer/node_modules/isbn/isbn.js:227:17
    at Object.<anonymous> (/opt/data-importer/node_modules/isbn/isbn.js:229:2)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/opt/data-importer/libs/rawBooks.js:6:14)
    at Module._compile (module.js:434:26)
```

I get this error in node 4.1

**Comment by CristyTarantino** [comment]
+1

**Comment by taitems** [comment]
@m-abs @CristyTarantino Howdy, this should be resolved now as part of #9. It was fixed and never deployed via an `npm publish`. Marking as resolved.
